R/yahoo_franchises.R

```diff
@@ -0,0 +1,55 @@
+#' Get a dataframe of franchise data
+#'
+#' @param conn a conn object created by `ff_connect()`
+#'
+#' @examples
+#' \donttest{
+#' try({ # try only shown here because sometimes CRAN checks are weird
+#'   yahoo_conn <- ff_connect(platform = "yahoo", league_id = "77275", token = NULL)
+#'   ff_userleagues(yahoo_conn)
+#' }) # end try
+#' }
+#' @describeIn ff_franchises Yahoo: Returns franchise data.
+#' @export
+ff_franchises.yahoo_conn <- function(conn) {
+  response <- yahoo_getendpoint(conn, glue::glue("leagues;league_keys={get_league_key(conn)}/teams"))
+  xml_doc <- xml2::read_xml(response$response$content)
+  xml2::xml_ns_strip(xml_doc)
+
+  # Extract franchise data
+  franchise_id <- as.integer(xml2::xml_text(xml2::xml_find_all(xml_doc, "//manager/manager_id")))
```

Comment:
Okay cool.  I didn't add the helpers just yet since some places I use find_all and some places I use find_first.  I updated to use the pipe syntax in most places, let me know if I went overkill on the pipes.